modules/control/controller/lon_controller.cc

```diff
@@ -295,13 +299,15 @@ Status LonController::ComputeControlCommand(
   }
 
   if (calibration_value >= 0) {
-    throttle_cmd = std::abs(calibration_value) > throttle_deadzone
+    throttle_cmd = std::abs(calibration_value) >
+                           std::max(throttle_deadzone, throttle_minimum_action)
                        ? std::abs(calibration_value)
                        : throttle_deadzone;
```

Comment:
You mean `max(std::abs(calibration_value), throttle_deadzone, throttle_minimum_action)`? The new code may result in `throttle_cmd < throttle_minimum_action` if `throttle_deadzone < throttle_minimum_action > std::abs(calibration_value)`.

Reply:
yep, simply choose the max of these three.

Reply:
Discuss with Qi. Logic modified. Thanks.